src/MigrateCommand.php

```diff
@@ -219,7 +219,6 @@ private function updatePackages(array $packages) : void
     {
         exec('rm -Rf vendor');
         exec('composer install --no-interaction');
-        exec('composer require "zendframework/zend-diactoros:^1.7.1"');
```

Comment:
We had issues with this before. Without this line it didn't work.

/Cc mwop what did you try last week and would this work?

Reply:
@xtreamwayz Please note I add it below in `$require` variable.
This is in case we have explicitly defined zend-diactoros in composer, then it was updated here, and removed later on (in loop with commands, because it's one of expressive deps - defined on top of the file). We have to install it always on migration, for compatibility, but it should be done later - after removing packages.

Reply:
I saw you added it. Just want to check with @weierophinney if he tried this solution and if it failed.

Reply:
I did indeed try adding it to the `require` section (that was actually my first attempt), and it did not work. I found I had to do the explicit `composer require` above for it to work correctly.

Reply:
Okay, so tried this locally; difference must be with the order in which the dev/non-dev requirements are installed, as it worked fine for me, both when used as a dev dependency of the project as well as used from a global installation.